src/atoms/Size/Size.web.js

```diff
@@ -1,3 +1,33 @@
-const Size = () => null;
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { makePxValue } from '../../_helpers/theme';
+
+const Size = styled(
+  ({ minWidth, width, maxWidth, minHeight, height, maxHeight, children, ...props }) => {
+    if (React.Children.toArray(children).length !== 1) {
+      throw new Error('Expected a single child for Size component');
+    }
+
+    return React.cloneElement(children, props);
+  },
+)`
+  ${(props) => (props.width ? `width: ${makePxValue(props.width)}` : '')};
+  ${(props) => (props.height ? `height: ${makePxValue(props.height)}` : '')};
+  ${(props) => (props.minWidth ? `min-width: ${makePxValue(props.minWidth)}` : '')};
+  ${(props) => (props.maxWidth ? `max-width: ${makePxValue(props.maxWidth)}` : '')};
+  ${(props) => (props.minHeight ? `min-height: ${makePxValue(props.minHeight)}` : '')};
+  ${(props) => (props.maxHeight ? `max-height: ${makePxValue(props.maxHeight)}` : '')};
+`;
+
+Size.propTypes = {
+  width: PropTypes.string,
```

Comment:
All the props should accept a string or number. String for absolute value and number for defining scale.
`width: PropTypes.oneOfType([PropTypes.number, PropTypes.string])`

Reply:
what about px or % ? @shubhnik

Reply:
That would just be strings.

Here the three possibilities:
`<Size height="20px"/>` (typeof height is string)
`<Size height="100%"/>` (typeof height is string)
`<Size height={2}/>` (typeof height is number)

Reply:
if 100 is passed as width, will it be 100px or 100%?

Reply:
if you just pass 
```
<Size height={100}/>
```
it will pass it through makePxValue function which then multiplies it with 8px grid and make it 100*8=800px

Check the native implementation and also the makePxValue function